README.md

```diff
@@ -1 +1,25 @@
-# ios-sdk-xcframeworks
\ No newline at end of file
+# Unblu iOS mobile SDK Swift Package
+
+The Unblu iOS mobile SDK serves as a way to integrate the Unblu platform into native iOS apps.
+
+Unblu delivers the Unblu iOS mobile SDK as a Swift package on GitHub: https://github.com/unblu/ios-sdk-xcframeworks
+
+# Installing
+
+To add the Unblu iOS mobile SDK Swift package to your Xcode project:
+
+1. In Xcode, go to **File** > **Add Packages**.
+2. Enter the following repository URL: https://github.com/unblu/ios-sdk-xcframeworks
+3. In the **Dependency Rule** dropdown, select **Up To Next Major Version**.
+4. Select the project you want to add it to.
+5. Click **Add Package**.
+6. In the window that appears, choose the **UnbluMobileSDK** package product and click **Add package**.
```

Comment:
There will be several packages here, they must select the necessary ones, then they will appear....

Reply:
What are the necessary packages? Ideally, we should list them.